lib/cli/commands/run.js

```diff
@@ -49,7 +49,7 @@ module.exports = run = {
             process.env.SAUCE_LABS = 'true';
             process.env.TUNNEL_IDENTIFIER = process.env.TUNNEL_IDENTIFIER || `MBTT${uuidv4()}`;
             if (configFile.saucelabs) {
-                process.env.SAUCE_CONFIG = JSON.stringify(saucelabs);
+                process.env.SAUCE_CAPABILITIES = JSON.stringify(saucelabs);
```

Comment:
sauce labs in the documentation is saying if its set to true or false... should it be if configFile.sauceCapabilities.

so in my config i would have {
saucelabs: true,
sauceCapabilities: {
some stuff
}

}

Reply:
also you cannot do if (configFile.sauceLabs) because configFile is not always defined if one is not specified